.github/workflows/test-sarif.yaml

```diff
@@ -0,0 +1,61 @@
+name: Test kube-linter SARIF output
+
+on:
+  - push
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          # Checkout all repo history to make tags available for figuring out kube-linter version during build.
+          fetch-depth: 0
+
+      - name: Read Go version from go.mod
+        run: echo "GO_VERSION=$(grep -E "^go\s+[0-9.]+$" go.mod | cut -d " " -f 2)" >> $GITHUB_ENV
+      - name: Setup Go environment
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.GO_VERSION }}
+
+      - name: Cache Go modules
+        uses: actions/cache@v2
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+
+      - name: Build binaries
+        run: make build
+
+      - name: Print kube-linter version
+        run: .gobin/kube-linter version
+
+      - name: Run kube-linter on a sample file with SARIF output
+        run: .gobin/kube-linter lint --format=sarif e2etests/testdata/inputs/splunk.yaml > results.sarif
+        continue-on-error: true
+
+      - name: Dump output file and check it is not empty
+        # The if part will generate no-zero exit code if the file is empty. See https://github.com/stedolan/jq/issues/1142#issuecomment-432003984
+        run: jq -es 'if . == [] then null else .[] | . end' results.sarif
+
+      - name: Upload output file as GitHub artifact for manual investigation
+        uses: actions/upload-artifact@v2
+        with:
+          name: results.sarif
+          path: results.sarif
+
+      - name: Check if output file is valid according to SARIF schema
+        run: |
+          set -ex
+          GO111MODULE=on go get github.com/neilpa/yajsv@v1.4.0
+          schema=$(jq -r '.["$schema"]' results.sarif)
+          [ -n "$schema" ] && wget -nv $schema
+          yajsv -s $(basename $schema) results.sarif
+
+      - name: Upload SARIF output file to GitHub
```

Comment:
Is this something we want to do on every commit? Seems like it was useful for your testing, but in steady state I don't think it makes sense?

Reply:
Hmm, okay with me. We can always remove it later if it seems noisy.